WordPress/Classes/ViewRelated/Reader/Cards/ReaderGapMarkerCell.xib

```diff
@@ -11,27 +13,34 @@
-                <frame key="frameInset" width="320" height="61"/>
+                <rect key="frame" x="0.0" y="0.0" width="320" height="62"/>
                 <autoresizingMask key="autoresizingMask"/>
                 <subviews>
                     <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="mwJ-ag-abH">
+                        <rect key="frame" x="-8" y="26.666666666666668" width="336" height="9.0000000000000036"/>
                         <constraints>
                             <constraint firstAttribute="height" constant="9" id="X0y-jv-BI5"/>
                         </constraints>
                     </view>
                     <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="kzd-gs-nwM">
+                        <rect key="frame" x="90" y="10" width="140" height="42"/>
                         <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                     </view>
                     <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Iu5-0g-eH5">
+                        <rect key="frame" x="90" y="10" width="140" height="42"/>
                         <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                     </view>
-                    <activityIndicatorView hidden="YES" opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" style="white" translatesAutoresizingMaskIntoConstraints="NO" id="7AJ-lZ-zT3"/>
+                    <activityIndicatorView hidden="YES" opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" style="white" translatesAutoresizingMaskIntoConstraints="NO" id="7AJ-lZ-zT3">
+                        <rect key="frame" x="150" y="21" width="20" height="20"/>
+                    </activityIndicatorView>
                     <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="gYN-FK-C3p">
+                        <rect key="frame" x="0.0" y="0.0" width="320" height="62"/>
                         <subviews>
-                            <button opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="1000" verticalCompressionResistancePriority="1000" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="srv-7s-DDa" customClass="PostMetaButton">
+                            <button opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="1000" verticalCompressionResistancePriority="1000" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="srv-7s-DDa">
```

Comment:
This is safe because `PostMetaButton` takes `imageEdgeInsets` and `titleEdgeInsets` into account, and `ReaderGapMarketCell` only changes `contentEdgeInsets`.

```swift
extension PostMetaButton {


     open override var intrinsicContentSize: CGSize {
         var newSize = super.intrinsicContentSize
         newSize.width += (imageEdgeInsets.left + imageEdgeInsets.right)
         newSize.width += (titleEdgeInsets.left + titleEdgeInsets.right)
         return newSize
     }
 }
```

<img width="320" alt="Screenshot 2025-04-22 at 1 20 40 PM" src="https://github.com/user-attachments/assets/a14bdf88-cd1d-4005-980c-7d35e3456af7" />